precompile/allow_list.go

```diff
@@ -59,55 +61,44 @@ func (c *AllowListConfig) Equal(other *AllowListConfig) bool {
 	if other == nil {
 		return false
 	}
-	if len(c.AllowListAdmins) != len(other.AllowListAdmins) {
+	if !checkList(c.AllowListAdmins, other.AllowListAdmins) {
+		return false
+	}
+
+	return checkList(c.EnabledAddresses, other.EnabledAddresses)
+}
+
+func checkList(current []common.Address, other []common.Address) bool {
```

Comment:
Can we add a comment and update the name to `isEqualAddressLists` ? Check leaves it a bit vague as to what is being checked.

Reply:
done